suite-native/module-trading/src/components/general/MyAssetSheet/MyAssetListItem.tsx

```diff
@@ -0,0 +1,81 @@
+import { useSelector } from 'react-redux';
+
+import { TradingRootState, selectTradingCoinInfoByCryptoId } from '@suite-common/trading';
+import { Account } from '@suite-common/wallet-types';
+import { VStack } from '@suite-native/atoms';
+import {
+    BaseCurrencyAmountFormatter,
+    CryptoAmountFormatter,
+    TokenAmountFormatter,
+} from '@suite-native/formatters';
+
+import { MyAsset, TradeableAsset } from '../../../types/general';
+import { coinInfoToTradeableAsset } from '../../../utils/general/tradeableAssetUtils';
+import { ASSET_ITEM_HEIGHT, AssetListItem } from '../AssetListItem';
+
+export type MyAssetListItemProps = {
+    account: Account;
+    asset: MyAsset;
+    onPress: (asset: TradeableAsset) => void;
+};
+
+export { ASSET_ITEM_HEIGHT };
+
+export const MyAssetListItem = ({ asset, onPress }: MyAssetListItemProps) => {
+    const { symbol, name, balance, fiatBalance, tokenSymbol, contract, cryptoId } = asset;
+
+    const coinInfo = useSelector((state: TradingRootState) =>
+        selectTradingCoinInfoByCryptoId(state, cryptoId),
+    );
+
+    const tradeableAsset = cryptoId && coinInfo && coinInfoToTradeableAsset(cryptoId, coinInfo);
+
+    if (!tradeableAsset) {
+        return null;
+    }
+
+    const balanceContent = (
+        <VStack alignItems="flex-end">
+            {tokenSymbol !== undefined ? (
+                <TokenAmountFormatter
+                    value={balance}
+                    tokenSymbol={tokenSymbol}
+                    numberOfLines={1}
+                    ellipsizeMode="tail"
+                    color="textDefault"
+                    variant="body"
+                />
+            ) : (
+                <CryptoAmountFormatter
+                    value={balance}
+                    symbol={symbol}
+                    numberOfLines={1}
+                    adjustsFontSizeToFit
+                    color="textDefault"
+                    variant="body"
+                />
+            )}
+
+            {fiatBalance && (
+                <BaseCurrencyAmountFormatter
+                    symbol={symbol}
+                    value={fiatBalance}
+                    variant="hint"
+                    color="textSubdued"
+                />
+            )}
+        </VStack>
+    );
+
+    return (
+        <AssetListItem
+            name={name}
+            symbol={symbol}
```

Comment:
using symbol means networks symbol, but we want asset symbol instead. Otherwise it will show BASE for WETH on Base

```suggestion
            symbol={tradeableAsset.symbol}
```

Reply:
Fixed in e509937bd1